### PR TITLE
feat(install): Require powershell to be run as admin

### PR DESCRIPTION
**Pull request by natecanfield822**
This adds the requires tag for admin for powershell.

### CODE REVIEW

**Comment by natecanfield822** [comment]
If you're not logged in as an admin and you don't run the script as an admin, the user will get the below error.

image

**Comment by Julien4218** [comment]
@natecanfield822 this is good, but perhaps not enough.
If a newrelic-cli was previously installed, our copy/paste continue to runs and execute the cli, here is an illustration
image

**Review by Julien4218 — CHANGES_REQUESTED** [review]
Good but not enough

**Comment by natecanfield822** [comment]
Unfortunately it looks like that requires syntax won't stop further command execution, only prevent the script itself from doing anything at all.

This should cause command execution to stop at the script if not run as an admin

File: scripts/install.ps1 [hunk 1]
```diff
@@ -1,3 +1,7 @@
+$p = New-Object System.Security.Principal.WindowsPrincipal([System.Security.Principal.WindowsIdentity]::GetCurrent())
+if (!$p.IsInRole([System.Security.Principal.WindowsBuiltInRole]::Administrator)) {
+    throw 'This script requires admin privledges to run and the current Windows PowerShell session is not running as Administrator. Start Windows PowerShell by  using the Run as Administrator option, and then try running the script again.'
```
Comment (Julien4218) [inline scripts/install.ps1:3]: @natecanfield822 this seems to work well, however could you double check the wording, for example privileges is misspell. There are also more than 1 spaces between words sometimes.

**Comment by Julien4218** [comment]
It works great, thanks @natecanfield822 
image

**Review by Julien4218 — APPROVED** [review]

**Pull request merged**